python/ray/data/_internal/compute.py

```diff
@@ -50,7 +50,6 @@ def __eq__(self, other: Any) -> bool:
         )
 
 
-@PublicAPI
```

Comment:
ooc, I thought we say every API symbol needs to get an annotation?

Reply:
except for everything inside the `_internal` package and those with with `_` in its name